source/dataman/HDF5/AMHDF5Error.cpp

```diff
@@ -0,0 +1,68 @@
+#include "AMHDF5Error.h"
+
+#define AMHDF5ERROR_ERROR_STACK_NOT_AQUIRED 700000
+#define AMHDF5ERROR_PRINT_FAILED 700001
+#define AMHDF5ERROR_CLEAR_FAILED 700002
+
+AMHDF5Error::AMHDF5Error(QObject *parent) : QObject(parent)
+{
+
+}
+
+AMHDF5Error::~AMHDF5Error()
+{
+
+}
+
+bool AMHDF5Error::printErrorStack()
+{
+	hid_t errorStack = H5Eget_current_stack();
+
+	if(errorStack < 0){
+		AMErrorMon::alert(this, AMHDF5ERROR_ERROR_STACK_NOT_AQUIRED, QString("Print: HDF5 error-stack could not be aquired."));
+		return false;
+	}
+
+	herr_t printStatus = H5Eprint2(errorStack, stdout);
```

Comment:
Is it all possible to be able to provide some kind of `FILE *` that would allow you to get this information into a QString to be added to an error monitor at all?